language/src/technique.rs

```diff
@@ -200,8 +202,36 @@ enum MethodElem {
     MethodBlock(MethodBlock),
 }
 
+#[derive(Serialize, Deserialize)]
+#[serde(rename_all = "lowercase")]
+enum ReportingLogicType {
+    Focus,
```

Comment:
what does the "focus" type does?

Reply:
Ask the author :-)
I means : "look and the value element to extract the reporting of a single sub item"

Reply:
so this one needs to be put on one of the states inside of the technique?

Reply:
This one is the technique editor representation of this information, but you are right, it should be a metadata of a state inside the block in the language

Reply:
LGTM with a comment explaining the focus case requires additional stuff

Reply:
I think it should be represented differently.

Focus should always have a "value" parameter and the other should never have one. I think their type should represent that

Reply:
More like 
enum ReportingLogic {
    Focus { value : String },
   Worst,
  Sum
}

Reply:
indeed

Reply:
but this woudn't match the json structure

Reply:
I think it should match the semantic of the program we want not the json structure, we don't want a value for Sum or Worst,and we always want one for Focus (where missing Value is an error and not optionnal)